src/components/Disputes/DisputeInfo.js

```diff
@@ -116,20 +115,7 @@ function DisputeHeader({ dispute, error }) {
           width: 100%;
         `}
       >
-        <div
-          css={`
-            background: linear-gradient(
-              233deg,
-              ${theme.accentStart} -50%,
-              ${theme.accentEnd} 91%
-            );
-            border-radius: 50%;
-            padding: ${1.5 * GU}px;
-            display: inline-block;
-          `}
-        >
-          <img src={iconCourt} alt="" width="39" height="32" />
-        </div>
+        <img src={iconDispute} alt="" width="70" height="70" />
```

Comment:
If it's not too much trouble, what does this look like? I'm just a bit worried on this one because it looks like we've 2x'd the size of this image.

Reply:
Ah this is the icon shown in the preview (Dispute detail, left top corner).
I should have clarified about the 2x in size, we exported the icons with the background integrated now.